term_posix.go

```diff
@@ -83,6 +84,52 @@ func (t *Term) setSpeed(baud int) error {
 	return termios.Tcsetattr(uintptr(t.fd), termios.TCSANOW, (*syscall.Termios)(&a))
 }
 
+func clamp(v, lo, hi int64) int64 {
+	if v < lo {
+		return lo
+	}
+	if v > hi {
+		return hi
+	}
+	return v
+}
+
+// timeoutVals converts d into values suitable for termios VMIN and VTIME ctrl chars
+func timeoutVals(d time.Duration) (uint8, uint8) {
+	if d > 0 {
+		// VTIME is expressed in terms of deciseconds
+		vtimeDeci := d.Nanoseconds() / 1e6 / 100
+		// ensure valid range
+		vtime := uint8(clamp(vtimeDeci, 1, 0xff))
+		return 0, vtime
+	} else {
```

Comment:
Please drop the else block, the if block always ends in a return so else is not needed